icechunk-python/src/config.rs

```diff
@@ -70,18 +71,62 @@ impl PyS3StaticCredentials {
         session_token = None,
         expires_after = None,
     ))]
-    fn new(
+    pub fn new(
         access_key_id: String,
         secret_access_key: String,
         session_token: Option<String>,
         expires_after: Option<DateTime<Utc>>,
     ) -> Self {
         Self { access_key_id, secret_access_key, session_token, expires_after }
     }
+
+    pub fn __repr__(&self) -> String {
+        // TODO: escape
+        format!(
+            r#"S3StaticCredentials(access_key_id="{ak}", secret_access_key="{sk}", session_token={st}, expires_after={ea})"#,
```

Comment:
do we want this? A user might end up printing this in a notebook and committing it... Since it's static the expiration time might be quite long.

Reply:
maybe we replace with `*`?